src/languages/es.js

```diff
@@ -352,4 +352,15 @@ export default {
         confirmCompanyIsNot: 'Confirmo que esta empresa no está en el',
         listOfRestrictedBusinesses: 'lista de negocios restringidos',
     },
+    requestorStep: {
```

Comment:
Posted in #espanol to get some help for this. I trust Google translate... but not that much 😅